yaml-tests/src/test/resources/valid-identifiers.yamsql

```diff
@@ -40,11 +40,11 @@ schema_template:
     create table "foo.tableE"("foo.tableE.E1" bigint, "foo.tableE.E2" bigint array, "foo.tableE.E3" "foo.struct", primary key("foo.tableE.E1"))
 
     create table "my$adjacency$list"("me" bigint, "my__parent" bigint, primary key("me"))
-    create function "__$func1" ( in "__$func1$A" bigint, in "__$func1$B" bigint )
-       as select "me" as "__1", "my__parent" as "__2" from "my$adjacency$list" where "me" < "__$func1$A" and "my$adjacency$list"."my__parent" = "__$func1$B"
-    create function "__$func2" ( "__$func2$A" bigint )
-       as select "me" as "__1", "my__parent" as "__2" from "my$adjacency$list" where "me" = "__$func2$A"
-    create function "__$func3" ( in "__$func3$A" bigint, in "__$func3$B" bigint, in "__$func3$C" bigint) as select "...f1"."__1", "...f1"."__2", "...f2"."__1", "...f2"."__2" from "__$func1"("__$func3$A", "__$func3$B") "...f1", "__$func2"("__$func3$C") "...f2"
+    create function "_$func1" ( in "_$func1$A" bigint, in "_$func1$B" bigint )
```

Comment:
It would be nice to revert as many of these changes as we can, I think, and validate that we get those doing the right thing. I think that seems like something we could achieve, and then the only changes in this file would be the negative tests and the `$$$$` to `A$$$$` changes